dev/sbt-scripted-tools/src/main/scala/com/lightbend/lagom/sbt/scripted/ScriptedTools.scala

```diff
@@ -110,10 +110,18 @@ object ScriptedTools extends AutoPlugin {
       file
     },
     aggregate in validateFile := false,
-    Internal.Keys.interactionMode := NonBlockingInteractionMode
-  )
-
-  override def projectSettings: Seq[Setting[_]] = Seq(
+    Internal.Keys.interactionMode := NonBlockingInteractionMode,
+    // This is copy & pasted from project/AkkaSnapshotRepositories so that scripted tests
+    // can use Akka snapshot repositories as well. If you change it here, remember to keep
+    // project/AkkaSnapshotRepositories in sync.
+    resolvers ++= (sys.env.get("TRAVIS_EVENT_TYPE").filter(_.equalsIgnoreCase("cron")) match {
+      case Some(_) =>
+        Seq(
+          "akka-snapshot-repository".at("https://repo.akka.io/snapshots"),
+          "akka-http-snapshot-repository".at("https://dl.bintray.com/akka/snapshots/")
+        )
+      case None => Seq.empty
+    }),
     scalaVersion := sys.props.get("scala.version").getOrElse("2.12.10")
```

Comment:
Should this be `scalaVersion in ThisBuild`?

The problem with Lagom `scripted` tests is that in some cases it's `sbt` single-module projects that happen to be a lagom service (e-g- `sbt-plugin/cross-build`) but then in other cases it's `sbt` multi-module projects.

For example, [this change](https://github.com/lagom/lagom/pull/2316/files#diff-0e641d6a07ee70570c339488fd2b1dd7L10) had a `commonSettings` defining `scalaVersion :=...` and then the `commonSettings were added to each of the lagom submodules.

Reply:
Yes, it should.  And by being in `buildSettings` it is.

Reply:
It is okay because it is now part of `buildSettings` instead of `projectSettings`. :-)